components/dashboards/org.wso2.carbon.dashboards.siddhi.apps.api/findbugs-exclude.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Why is he suppression file added? We are not suppressing anything

Reply:
Fixed in bbb8597